app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] I think it would be great if you add confirmable to your user model since I tried to comment on the validations to complete the signup but it let me sign without asking for confirmation so kindly make sure the user confirms his email before signing in 👏🏻⭐ so please add it since it is one of the requirements 🙏🏻